framework/UPGRADE.md

```diff
@@ -105,6 +105,8 @@ Upgrade from Yii 2.0.x
   extend `yii\caching\SimpleCache` abstract class. Use `yii\caching\CacheInterface` only if you wish to replace `yii\caching\Cache`
   component providing your own solution for cache dependency handling.
 * Console command used to clear cache now calls related actions "clear" instead of "flush".
+* Yii autoloader was removed in favor of Composer-generated one.
```

Comment:
Let's write explicitly what does user should do for this change. How about:

<code>
Yii autoloader was removed in favor of Composer-generated one. You should no longer write `require __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';` in your application entrypoint. Removed method `Yii::autoload()` and property `Yii::$classMap`.
</code>

Reply:
Done.